src/Bounds.cpp

```diff
@@ -645,14 +639,20 @@ class Bounds : public IRVisitor {
             min = Expr(); max = Expr(); return;
         }
 
+        bool const_condition = !expr_uses_vars(op->condition, scope);
```

Comment:
what happens if condition is constant but one of the values is undefined? should we return undefined or `select(cond, some value, undefined)`?

btw, line 438-440 on the Div handler
```
Expr cmp = min_b > make_zero(min_b.type().element_of());
min = select(cmp, a, b);
max = select(cmp, b, a);
```
This doesn't handle the case when b is zero, does it? 
`x/y (0 <= x <= 10)` will return `min: select((y > 0), (0/y), (10/y)); max: select((y > 0), (10/y), (0/y))`